amethyst_network/src/simulation/resource.rs

```diff
@@ -115,8 +117,9 @@ impl<S: Socket> NetworkSimulationResource<S> {
         payload: &[u8],
         delivery: DeliveryRequirement,
         timing: UrgencyRequirement,
+        stream_id: Option<u8>,
```

Comment:
Instead of the parameter, can we add it to `DeliveryRequirement` like : `DeliveryRequirement::ReliableOrdered(Option<u8>)`

Reply:
Yeah I guess it is only useful on ordered connections.

Reply:
Actually, to that point, why do we have a stream option on `sequenced` packets in Laminar?

Reply:
Because ordering and sequencing are done on separate streams. And by default 255 or something like that is used, unless specifying `Some(id)`

Reply:
What I mean is, what problem is the stream_id solving for sequenced messages? For ordering it makes sense because we don't want some messages to wait to be delivered if received out of order. However, nothing waits on sequenced messages so as long as they are handled separately from the Ordered messages they shouldn't need a separate logical stream right?

Reply:
I thought about it and answered my own question. Will asd 👍